public/index.js

```diff
@@ -3,31 +3,31 @@ import {mapAndInit} from "./integration.js";
 let cdeFile = null;
```

Comment:
@Aiga115 I'm not sure but I think public should not be in git. Can you check and if so remove it and add public to .gitignore ?
Thanks

Reply:
I see that when we run `npm run dev` the files served are the ones under the public folder (not the ones under demo). With such configuration, I would say it makes more sense to keep the public in git and remove the duplicated demo files but I might be missing something. What do you think @zsinnema?

Reply:
@afonsobspinto the demo is used in the Docker file.